modules/common/src/test/java/com/ibm/cloud/cloudant/internal/ValidationTest.java

```diff
@@ -14,16 +14,50 @@
 
 import java.util.Map;
 import java.util.Map.Entry;
+import java.util.concurrent.TimeUnit;
+
 import com.ibm.cloud.cloudant.common.SdkCommon;
+import com.ibm.cloud.sdk.core.http.HttpConfigOptions;
 import com.ibm.cloud.sdk.core.security.NoAuthAuthenticator;
+import okhttp3.OkHttpClient;
 import org.testng.annotations.Test;
+import static org.testng.Assert.assertEquals;
 import okhttp3.HttpUrl;
 import okhttp3.Request;
 
 public class ValidationTest {
     
     private final String dbName = "testDatabase";
     private final String docId = "_testDocument";
+
+    @Test
+    void validateDefaultReadTimeoutSetting() {
```

Comment:
Do you think this function should be renamed since we are also verifying the `callTimeout` option?

Reply:
I discussed this topic with Rich, considering there is already a default 60s read timeout available for authentication by `HTTPSinglton`, these client options do not need overwrite. 60s will be enough to handle the authentication requests.